abb_irb52_support/package.xml

```diff
@@ -34,14 +34,15 @@
   <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
 
   <buildtool_depend>catkin</buildtool_depend>
+  
   <build_depend>roslaunch</build_depend>
```

Comment:
<build_depend>roslaunch</build_depend> should be replaced with   <depend>roslaunch</depend>

Reply:
replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn